<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>H264</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: white;
	font-family: &quot;Segoe UI&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="base_frame">
     <property name="styleSheet">
      <string notr="true">QFrame#base_frame {
	background-color: #333333;
	border-radius: 10px;

}

QComboBox QAbstractItemView {
	color: black;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="title_bar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
	background-color: #f77f00;
	border-top-left-radius: 10px;
	border-top-right-radius: 10px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="title">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: white;
	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;

}</string>
           </property>
           <property name="text">
            <string>H264 SETUP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_title_bar">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>677</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="main_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>20</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QFrame" name="bitrate_mode_frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>110</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>110</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>5</number>
            </property>
            <property name="verticalSpacing">
             <number>10</number>
            </property>
            <item row="3" column="0">
             <widget class="QRadioButton" name="crf_radio">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton::indicator {
	background-color: #333333;
	border-radius: 6px;
	border: 1px solid #7f7f7f;
	width: 12px;
	height: 12px;
}

QRadioButton::indicator:hover {
	border: 1px solid #f77f00;
}

QRadioButton::indicator:pressed {
	border: 1px solid #f77f00;
	background-color: #222222;
}

QRadioButton::indicator:checked {
	image: url(:/icons/icons/check_white_18dp.svg)
}
</string>
              </property>
              <property name="text">
               <string>CRF</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="bitrate_title">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>15</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>15</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
	font-weight: bold

}</string>
              </property>
              <property name="text">
               <string>Bitrate mode</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QRadioButton" name="vbv_radio">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton::indicator {
	background-color: #333333;
	border-radius: 6px;
	border: 1px solid #7f7f7f;
	width: 12px;
	height: 12px;
}

QRadioButton::indicator:hover {
	border: 1px solid #f77f00;
}

QRadioButton::indicator:pressed {
	border: 1px solid #f77f00;
	background-color: #222222;
}

QRadioButton::indicator:checked {
	image: url(:/icons/icons/check_white_18dp.svg)
}
</string>
              </property>
              <property name="text">
               <string>VBV</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="crf_spinbox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QSpinBox {
	background-color: #333333;
	color: white;
	border: 1px solid #7f7f7f;
	border-radius: 5px;
	padding: 2px;
	padding-left: 4px;

}

QSpinBox:hover {
	border: 1px solid #f77f00;
}

QSpinBox:focus {
	border: 1px solid #f77f00;
}

QSpinBox:disabled {
	background-color: #555555;
	color: #262626;
	border-color: rgb(80, 80, 80);
}

</string>
              </property>
              <property name="maximum">
               <number>51</number>
              </property>
              <property name="value">
               <number>17</number>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="vbv_input">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
	background-color: #333333;
	color: white;
	border: 1px solid #7f7f7f;
	border-radius: 5px;
	padding: 2px;

}

QLineEdit:hover {
	border: 1px solid #f77f00;
}

QLineEdit:focus {
	border: 1px solid #f77f00;
}

QLineEdit:disabled {
	background-color: #555555;
	color: #262626;
	border-color: rgb(80, 80, 80);

}

</string>
              </property>
              <property name="placeholderText">
               <string>5000k</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QRadioButton" name="cbr_radio">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton::indicator {
	background-color: #333333;
	border-radius: 6px;
	border: 1px solid #7f7f7f;
	width: 12px;
	height: 12px;
}

QRadioButton::indicator:hover {
	border: 1px solid #f77f00;
}

QRadioButton::indicator:pressed {
	border: 1px solid #f77f00;
	background-color: #222222;
}

QRadioButton::indicator:checked {
	image: url(:/icons/icons/check_white_18dp.svg)
}
</string>
              </property>
              <property name="text">
               <string>CBR</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="cbr_input">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
	background-color: #333333;
	color: white;
	border: 1px solid #7f7f7f;
	border-radius: 5px;
	padding: 2px;

}

QLineEdit:hover {
	border: 1px solid #f77f00;
}

QLineEdit:focus {
	border: 1px solid #f77f00;
}

QLineEdit:disabled {
	background-color: #555555;
	color: #262626;
	border-color: rgb(80, 80, 80);

}

</string>
              </property>
              <property name="placeholderText">
               <string>5000k</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QFrame" name="other_frame">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>5</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="preset_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Preset</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="tune_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Tune</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="preset_combo">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox {
	background-color: #333333;
	color: white;
	border: 1px solid #7f7f7f;
	border-radius: 5px;
	padding: 2px;
	padding-left: 4px;

}

QComboBox:hover {
	border: 1px solid #f77f00;
}

QComboBox:focus {
	border: 1px solid #f77f00;
}
</string>
              </property>
              <property name="currentIndex">
               <number>5</number>
              </property>
              <item>
               <property name="text">
                <string>ultrafast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>superfast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>veryfast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>faster</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>fast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>medium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>slow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>slower</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>veryslow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>placebo</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="tune_combo">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox {
	background-color: #333333;
	color: white;
	border: 1px solid #7f7f7f;
	border-radius: 5px;
	padding: 2px;
	padding-left: 4px;

}

QComboBox:hover {
	border: 1px solid #f77f00;
}

QComboBox:focus {
	border: 1px solid #f77f00;
}

QComboBox::drop-down {
	background: none;
	outline: none;
}</string>
              </property>
              <item>
               <property name="text">
                <string>none</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>film</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>animation</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>grain</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>stillimage</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>fastdecode</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>zerolatency</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>psnr</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ssim</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="confirm_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: white;
	background-color: none;
	border-radius: 7px;
	border: 1px solid #7f7f7f;
	padding: 0px;
	font-weight: bold;
}

QPushButton:hover {
	border: 1px solid rgb(33, 213, 93);
}

QPushButton:pressed {
	background-color: rgb(33, 213, 93);
}



</string>
           </property>
           <property name="text">
            <string>Confirm</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resources.qrc">
             <normaloff>:/icons/icons/check_white_18dp.svg</normaloff>:/icons/icons/check_white_18dp.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>crf_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>crf_spinbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>crf_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>vbv_input</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>crf_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbr_input</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vbv_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>crf_spinbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vbv_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>vbv_input</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vbv_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbr_input</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbr_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>crf_spinbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbr_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>vbv_input</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbr_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbr_input</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
